cli/src/main/scala/filodb.cli/CliMain.scala

```diff
@@ -165,39 +173,40 @@ object CliMain extends FilodbClusterNode {
           promFilterToPartKeyBr(args.promql(), args.schema())
 
         case Some("partKeyBrAsString") =>
-          require(args.hexPk.isDefined, "--hexPk must be defined")
-          partKeyBrAsString(args.hexPk())
+          require(args.hexpk.isDefined, "--hexPk must be defined")
+          partKeyBrAsString(args.hexpk())
 
         case Some("decodeChunkInfo") =>
-          require(args.hexChunkInfo.isDefined, "--hexChunkInfo must be defined")
-          decodeChunkInfo(args.hexChunkInfo())
+          require(args.hexchunkinfo.isDefined, "--hexChunkInfo must be defined")
+          decodeChunkInfo(args.hexchunkinfo())
 
         case Some("decodeVector") =>
-          require(args.hexVector.isDefined && args.vectorType.isDefined, "--hexVector and --vectorType must be defined")
-          decodeVector(args.hexVector(), args.vectorType())
+          println("inside decodeVector "+args.hexvector)
```

Comment:
println? Leftover debug?

Reply:
Yes this and one other println was left behind. Fixed it.